Remove further mentions of the bootstrap committee from GC charter (#1837)
open-telemetry/community: governance-charter.md

- Representatives from major organizations that have adopted OpenTelemetry as
  end-users (one representative per organization)

The Governance Committee explicitly believes that this heuristic will be
inaccurate and not represent the entire community. An exception form will be
provided for people who have contributed to the project but may not meet these
criteria. Exception eligibility applications will be approved by the Governance
Committee by a simple majority vote. The exception process will be used as data
points for refining the criteria for the future.

> It is the responsibility of the Governance Committee to refine these criteria prior to each election, including adjusting the number of required contributions, and adding any additional community memberships that include eligibility.

Author: ```suggestion
It is the responsibility of the Governance Committee to refine these criteria 
prior to each election, including adjusting the number of required contributions
, and adding any additional community memberships that include eligibility.
```

- Representatives from major organizations that have adopted OpenTelemetry as
  end-users (one representative per organization)

The Governance Committee explicitly believes that this heuristic will be
inaccurate and not represent the entire community. An exception form will be
provided for people who have contributed to the project but may not meet these
criteria. Exception eligibility applications will be approved by the Governance

> committee by a simple majority vote. The exception process will be used as data

Reviewer: ```suggestion
Committee by a simple majority vote. The exception process will be used as data
```

Author: TY!

points for refining the criteria for the future.

> The Governance Committee commits to redefine these criteria around a more robust 

Reviewer 2: IMO, what we have is robust enough, and providing the exception form has proven to be quite sufficient. I'd be more comfortable if we stated that we'd be re-evaluating the current criteria by the end of 2024.

Author: This would work for me as well, btw, k8s uses this wording

> It is the responsibility of the steering committee to refine these criteria prior to each election, including setting the number of required contributions, and adding any additional committee memberships that include eligibility.

I was thinking about an adjusted version of that

Reviewer 2: Sounds good to me.

Author: updated, see the new version

I kept the piece with the "setting the number of required contributions", because at some point we _may_ consider raising that value above 20. I replaced the word "committee memberships" with "community membership" since this is referring to another sentence in the k8s document around code of conduct and other committees we do not have as of today.

> It is the responsibility of the Governance Committee to refine these criteria 

Reviewer 3: unnecessary space at the end.

Author: Ah, thanks for catching this

> It is the responsibility of the Governance Committee to refine these criteria 

Author: ```suggestion
It is the responsibility of the Governance Committee to refine these criteria
```

> The Governance Committee commits to redefine these criteria around a more robust 

Reviewer: ```suggestion
The Governance Committee commits to refine these criteria around a more robust 
```

> The Governance Committee commits to redefine these criteria around a more robust 
> idea of community membership by the end of 2024.

Author: ```suggestion
It is the responsibility of the Governance Committee to refine these criteria prior to each election, including adjusting the number of required contributions, and adding any additional community memberships that include eligibility.
```

- Representatives from major organizations that have adopted OpenTelemetry as
  end-users (one representative per organization)

The Governance Committee explicitly believes that this heuristic will be
inaccurate and not represent the entire community. An exception form will be
provided for people who have contributed to the project but may not meet these
criteria. Exception eligibility applications will be approved by the Governance
Committee by a simple majority vote. The exception process will be used as data
points for refining the criteria for the future.

> It is the responsibility of the Governance Committee to refine these criteria 
> prior to each election, including adjusting the number of required contributions,
> and adding any additional community memberships that include eligibility.

Reviewer: I'm not clear what this means

> adding any additional community memberships that include eligibility

maybe can simplify

```suggestion
It is the responsibility of the Governance Committee to refine these criteria 
prior to each election.
```

Author: yes, that works even better, thanks!

> prior to each election, including adjusting the number of required contributions
> , and adding any additional community memberships that include eligibility.

Author: ```suggestion
prior to each election, including adjusting the number of required contributions,
and adding any additional community memberships that include eligibility.
```
